src/transformers/models/detr/modeling_detr.py

```diff
@@ -1397,8 +1399,15 @@ def forward(
         >>> inputs = feature_extractor(images=image, return_tensors="pt")
         >>> outputs = model(**inputs)
         >>> # model predicts bounding boxes and corresponding COCO classes
-        >>> logits = outputs.logits
-        >>> bboxes = outputs.pred_boxes
+        >>> bboxes, logits = outputs.pred_boxes, outputs.logits
+
+        >>> probas_per_class = outputs.logits.softmax(-1)[:, :, :-1]
```

Comment:
I guess the last one is for `no-object`? Maybe a comment here would help the reader to understand the doc. easier 😄